features/dev/rni.feature

```diff
@@ -0,0 +1,44 @@
+# language: nl
+
+@post-assert
+Functionaliteit: RNI
+
+  Rule: wanneer er geen RNI-deelnemer is en er wordt niet om gevraagd met fields, wordt het niet geleverd
+
+    Scenario: geen RNI-deelnemer en niet vragen om rni
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000036                       |
+      | fields              | naam.voornamen                  |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En bevat het antwoord GEEN 'rni'
```

Comment:
Aanpassen naar dezelfde voorgestelde stap in ../rni.feature.

Is het niet handig om de nieuwe mappen structuur van historie te gebruiken? Dus rni map en daaronder dev sub-map met rni feature files?

Reply:
aangepast zoals voorgesteld: En heeft de persoon GEEN 'rni'

aanpassen van de mappenstructuur zou ik niet doen voor alleen rni, lijkt me handiger dat dan voor alle features te doen